src/Ext/freecad/CMakeLists.txt

```diff
@@ -4,6 +4,12 @@ OUTPUT_VARIABLE python_libs OUTPUT_STRIP_TRAILING_WHITESPACE )
 SET(PYTHON_MAIN_DIR ${python_libs})
 
 set(NAMESPACE_INIT "${CMAKE_BINARY_DIR}/Ext/freecad/__init__.py")
+if (WIN32)
+	set(FREECAD_LIBRARY_INSTALL_DIR ${CMAKE_INSTALL_BINDIR})
```

Comment:
@wwmayer do you know how to get the absolute-path here? CMAKE_INSTALL_BINDIR gives only "bin".

Reply:
To get an absolute path you have to use:
set(FREECAD_LIBRARY_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR})

FYI, for all systems other than Windows we use cmake's GNUInstallDirs module which defines the variables CMAKE_INSTALL_BINDIR, CMAKE_INSTALL_DATADIR, .. and they are all relative paths.
For Windows we define these variables the same way inside ConfigureCMakeVariables.cmake.